src/pages/ALLO_ManageAllocations.page

```diff
@@ -162,7 +160,6 @@
 
     <div class="slds-scope">
         <apex:form id="theForm">
-                <c:UTIL_NavigateBack recordId="{!parentId}" redirect="{!redirect}"/>
```

Comment:
@lparrott , @jbrandolinisf : can you please review and test the Manage Allocations page?

Reply:
@RadSFDeveloper - Looked over the PR this morning- looks great!  I am building a scratch org to do a few different tests, and then will approve!!!  Neat little addition to the manage allocations page!

Reply:
Thank you very much @jbrandolinisf  for reviewing the PR!